isb_lib/data_import/csv_import.py

```diff
@@ -59,14 +59,23 @@ def unflatten_csv_row(row: dict) -> dict:
         if k.startswith("producedBy"):
             flattened_row.pop(k)
             if "location" in k:
-                produced_by_sampling_site_location_dict[k.replace("producedBy_samplingSite_location_", "")] = v
+                k = k.replace("producedBy_samplingSite_location_", "")
+                if k == "elevationInMeters":
+                    k = "elevation"
```

Comment:
@datadavev this looks like a difference between SOLR and our JSON format.  I prefer the more verbose `elevationInMeters` which we are using in SOLR, but I think we left it generic because we might want to use different units at some point.